infra/conf/freedom.go

```diff
@@ -168,12 +152,15 @@ func (c *FreedomConfig) Build() (proto.Message, error) {
 		}
 	}
 	if c.Noise != nil {
-		for _, n := range c.Noise {
+		return nil, errors.New(`Freedom settings: please use "noises":[{}] instead of "noise":{}`)
```

Comment:
In order to maintain backwards compat, I suggest to append to `config.Noises` instead. The behavior when both are defined can be an error, I think.